service-front/module/Application/src/Controller/IndexController.php

```diff
@@ -7,9 +7,12 @@
 use Application\Contracts\OpgApiServiceInterface;
 use Laminas\Mvc\Controller\AbstractActionController;
 use Laminas\View\Model\ViewModel;
+use Application\Traits\FeatureCheck;
 
 class IndexController extends AbstractActionController
 {
+    use FeatureCheck;
```

Comment:
Rather than a trait, it might be easier to make this a [listener](https://docs.laminas.dev/laminas-mvc/examples/#registering-module-specific-listeners) so that it can naturally interrupt all requests to the front end. I'm similarly looking at [a listener to check auth](https://github.com/ministryofjustice/opg-paper-identity/pull/20/files).